libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/ForegroundService.kt

```diff
@@ -92,6 +97,10 @@ abstract class ForegroundService : SdkForegroundService, Service() {
      *  a ConnectRequest to be used to call the Breez SDK connect function. */
     abstract fun getConnectRequest(): ConnectRequest?
 
+    /** To be implemented by the application foreground service.
+     * Allows the user to override the default ExtensionSettings. */
+    abstract fun getExtensionSettings(): ExtensionSettings?
```

Comment:
NIT: I think the term extension is a bit confusing. Perhaps `getServiceConfig` or `getServiceSettings` ?